ovos_plugin_manager/wakewords.py

```diff
@@ -96,13 +96,17 @@ def load_module(module, hotword, config, lang, loop):
     @classmethod
     def create_hotword(cls, hotword="hey mycroft", config=None,
                        lang="en-us", loop=None):
-        config = get_hotwords_config(config)
-        config = config.get(hotword) or config["hey mycroft"]
-        module = config.get("module", "pocketsphinx")
+        ww_configs = get_hotwords_config(config)
+        ww_config = ww_configs.get(hotword) or ww_configs.get("hey mycroft")
+        module = ww_config.get("module", "pocketsphinx")
         try:
-            return cls.load_module(module, hotword, config, lang, loop)
+            return cls.load_module(module, hotword, ww_config, lang, loop)
         except:
-            LOG.error(f"failed to created hotword: {config}")
+            LOG.error(f"Failed to load hotword: {hotword} - {module}")
+            fallback_ww = ww_config.get("fallback_ww")
+            if fallback_ww in ww_configs and fallback_ww != hotword:
```

Comment:
If the fallback is also enabled, should we skip it?

What should happen if the user adds a `hey_mycroft_whisper` config that falls back to `hey_mycroft`; seems like this implementation would result in 2 precise instances if whisper fails and default config isn't overridden.

Reply:
Out of scope, this just loads the configured fallback and it's up to the calling module to deal with any other hotwords it wants to load